doc/source/tutorial/optimize.rst

```diff
@@ -1578,26 +1595,41 @@ We can solve the assignment problem with :func:`linear_sum_assignment`:
 
     >>> from scipy.optimize import linear_sum_assignment
     >>> row_ind, col_ind = linear_sum_assignment(cost)
+
+The ``row_ind`` and ``col_ind`` are optimal assigned matrix indexes of the cost matrix:
+
+::
+
+    >>> row_ind
+    array([0, 1, 2, 3])
     >>> col_ind
     array([0, 2, 3, 1])
 
 The optimal assignment is:
 
 ::
 
-    >>> dict(zip(["backstroke", "breaststroke", "butterfly", "freestyle"],
-    ...          np.array(["A", "B", "C", "D", "E"])[col_ind]))
+    >>> styles = np.array(["backstroke", "breaststroke", "butterfly", "freestyle"])[row_ind]
+    >>> students = np.array(["A", "B", "C", "D", "E"])[col_ind]
+    >>> dict(zip(styles, students))
     {'backstroke': 'A', 'breaststroke': 'C', 'butterfly': 'D', 'freestyle': 'B'}
 
-The optimal times for each swimming style and the total time are:
+The optimal total medley time is:
 
 ::
 
-    >>> cost[row_ind, col_ind]
-    array([43.5, 39.1, 44.5, 36.8])
     >>> cost[row_ind, col_ind].sum()
     163.89999999999998
 
+Note that this result is not the same as the sum of the minimum times for each swimming style:
+
+::
+
+    >>> np.min(cost, axis=1).sum()
+    161.39999999999998
+
+Because, student "C" is the best swimmer in both "breaststroke" and "butterfly" style.
+We need to decide either style is assigned to student "C" based on the total time.
```

Comment:
We cannot assign student "C" to both styles, so we assigned student C to the _____ style and ____ to the ____ style to minimize the total time.

Reply:
Thanks. I fixed.